exonum/src/lib.rs

```diff
@@ -22,9 +22,19 @@
 #![cfg_attr(
     feature = "cargo-clippy",
     allow(
-          stutter, similar_names, items_after_statements, use_self,
+          items_after_statements, option_map_unwrap_or_else,
+          // Next cast.. lints don't give alternatives.
           cast_possible_wrap, cast_possible_truncation, cast_sign_loss,
-          pub_enum_variant_names, single_match_else, option_map_unwrap_or_else
+          // The lint does not work properly with desugaring and macro.
+          used_underscore_binding,
```

Comment:
What king of desugaring? As for macros: I propose to suppress this lint locally (inside of macro or inside of modules with heavy macros usage).